.github/ISSUE_TEMPLATE/new-paper-template.md

```diff
@@ -9,8 +9,9 @@ assignees: ''
 
 <!--Hi there! Please use the template below as a guide for what information about this paper to include. It's ok to leave fields blank so that other contributors can fill them in later, or to add fields that you think are important.-->
 
-Title: Please edit the title to add the name of the paper after the colon
+<!--Title: Please edit the title to add the name of the paper after the colon.-->
```

Comment:
Did you mean for this to be a comment? I had changed it to not be a comment because I thought we wanted it to show up.

Reply:
Ah, I'm fine with either, but I interpreted it as something we meant as a comment (since I thought it was referring to the title of the issue)!

Reply:
Ahhhhh - I thought we wanted the title of the paper here. Well, then I broken the other ones 😂

Reply:
Let's leave it then! It wouldn't hurt to have it there.